test/e2e/preflightCliTest.go

```diff
@@ -0,0 +1,106 @@
+package e2e
+
+import (
+	"fmt"
+	"io"
+	"os/exec"
+)
+
+func preflightHelp() string {
+	cmd := exec.Command("preflight")
+	output, err := cmd.Output()
+
+	if err != nil {
+		fmt.Println("error:", err)
+	} else {
+		//#fmt.Printf("output: %s \n\n\n%s", string(output), string(want))
+	}
+	return string(output)
+}
+
+func preflightCheck() string {
+	cmd := exec.Command("preflight", "check")
```

Comment:
you better receive the path to the preflight binary from the env var, and give the end user to be in any dir when running the e2e check.